app/posts/publish-teacher-training-courses/2019-11-01-accredited-bodies-research-round-3.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Accredited bodies research - Round 3
+description: Research on the role of accredited bodies and their relationships with their partner schools
+date: 2019-11-01
+related:
+  items:
+  - text: Lookback sessions
+    href: https://lookback.io/dfe-digital/find-accrediting-bodies-round-3
+  - text: Research playback
+    href: https://docs.google.com/presentation/d/1tt_QwL2cU7fpZ9pg4JywENfJGp66DBWtkfTqiBYsLEY/edit#slide=id.g75932c5169_0_62
+---
+Round 3 is a continuation of the research started in [Accredited Bodies Research Round 1](/publish-teacher-training-courses/accredited-bodies-research-round-1) and [Round 2](/publish-teacher-training-courses/accredited-bodies-research-round-2). This round examines the role of accredited bodies and their relationships with their partner schools. 
+
+## Findings
+
+* At the start of each recruitment cycle, UCAS compiles list of reference data for any providers to consume. This is done via XML or ODBC into the provider’s SRS system. Any changes after this are not automatically synced.
+* Accrediting body organisations’ needs vary widely. HEIs and SCITTs that are part of HEIs have SRS. Other SCITTs will use UCAS Weblink and manual processes in order to process applications. Those with SRS are affected by systems not being in sync.
+* The staging area is an area within the SRS where applications sit to be processed. Applications are stuck in staging because the application cannot be matched to a course in the system. Providers get an import error list, where details about the candidate are present but nothing about the course.To fix this, they typically will log onto UCAS Weblink to find the application, note the details and then go onto Find to find the course to set up in their SRS.
+* Providers have very different student record systems - there is the base application but much of the features are personalised to the organisation.
+* Organisations map out codes themselves e.g. Location in School A on UCAS will be assigned to Code X in one organisation’s SRS and Code Z in a different organisation with the same location. This can make integration difficult.
+* Providers need CSVs for multiple reasons. They need to check against allocations spreadsheets, use spreadsheets to share data themsleves as there is no single access point and alos for data manipulation.
+
+### Why do accredited bodies need course information  
+
+_"We need to know because we are accountable for all the courses that we accredit. There is a reputational risk if we don’t have access [to the information]."_
```

Comment:
`>` will make a blockquote for quotes

```suggestion
> "We need to know because we are accountable for all the courses that we accredit. There is a reputational risk if we don’t have access [to the information]." 
```